src/test/java/org/apache/maven/plugins/install/stubs/InstallArtifactStub.java

```diff
@@ -85,8 +85,8 @@ public void addMetadata(ArtifactMetadata metadata) {
         }
     }
 
-    public Collection getMetadataList() {
-        return metadataMap == null ? Collections.EMPTY_LIST : metadataMap.values();
+    public Collection<ArtifactMetadata> getMetadataList() {
+        return metadataMap == null ? Collections.emptyList() : metadataMap.values();
```

Comment:
Isn*t this inconsistent? One is writable, the other one not?

Reply:
This is a method of a stub, and test is happy, IMHO is irrelevant.

Reply:
right